yarn-project/end-to-end/src/e2e_p2p/gossip_network.test.ts

```diff
@@ -75,7 +75,6 @@ describe('e2e_p2p_network', () => {
     t.logger.info('Creating nodes');
     nodes = await createNodes(
       t.ctx.aztecNodeConfig,
-      t.peerIdPrivateKeys,
```

Comment:
rediscovery test is not longer provided the same private keys, it is expected to store it on its own